rs/crypto/internal/crypto_service_provider/src/vault/remote_csp_vault/tarpc_csp_vault_client.rs

```diff
@@ -92,9 +93,26 @@ pub enum RemoteCspVaultError {
 }
 
 impl RemoteCspVault {
+    // This function panics if the calling stack enters (and remains within) a Tokio runtime.
     fn tokio_block_on<T: Future>(&self, task: T) -> T::Output {
         self.tokio_runtime_handle.block_on(task)
     }
+
+    // This is a safer version of the function above that blocks the local thread until the future is executed but
+    // it will not panic if the parent context is within a Tokio runtime.
+    // Use this function when you are unsure the calling stack enters (and remains within) a Tokio runtime.
```

Comment:
```suggestion
    /// This is a safer version of the function above that blocks the local thread until the future is executed but
    /// it will not panic if the parent context is within a Tokio runtime.
    /// Use this function when you are unsure the calling stack enters (and remains within) a Tokio runtime.
```